inspector/src/components/actionTabs/tabs/propertyGrids/animations/svgDraggableArea.tsx

```diff
@@ -71,7 +71,7 @@ export class SvgDraggableArea extends React.Component<ISvgDraggableAreaProps>{
             if (coord !== undefined) {
 
                 var newPoints = [...this.props.keyframeSvgPoints];
-
+                // Check for NaN values here. 

```

Comment:
Is that a TODO?

Reply:
Yes 😄 but I haven't been able to work on a bug that I'm guessing happens here. And also I don't have the exact steps to reproduce. So I added the comment here to work on later.